code/game/atoms.dm

```diff
@@ -1952,6 +1952,8 @@
 	return TRUE
 
 /atom/MouseEntered(location, control, params)
+	if(flags_1 & INITIALIZED_1)
+		usr.MouseEnteredOn(src, location, control, params)
```

Comment:
The proper pattern for this is a fullscreen object to act as a catcher. see `/atom/movable/screen/fullscreen/scope`
This ain't acceptable in any way, this proc is called for every object a client moves their mouse over in the space of a tick, 10 is maybe the worst case scenario

This code needs to be as lightweight as is possible.